payments/schema/mutations.py

```diff
@@ -356,6 +363,76 @@ def mutate_and_get_payload(cls, root, info, **input):
         return DeleteBerthProductMutation()
 
 
+class OrderLineInput:
+    quantity = graphene.Int(description="Defaults to 1")
+
+
+class CreateOrderLineMutation(graphene.ClientIDMutation):
+    class Input(OrderLineInput):
+        order_id = graphene.ID(required=True)
+        product_id = graphene.ID(required=True)
+
+    order_line = graphene.Field(OrderLineNode)
```

Comment:
hm, maybe it would make sense to also return the order as a separate field? since you can't traverse it back from the order line

Reply:
Hmmm true. We can add an `order` return field and remove later if its' not necessary. 👍

Reply:
But :big-hm:, for `createOrderLine`, `order` is a required field, so, would it make sense to also return it since you'd in theory know which order it belongs to?

Reply:
hm, well, you are kinda mutating Order by adding more lines to it and maybe you'll want to refresh some of the data displayed... 🤔  I see some potential use for it on the client's side, but in theory you could just resend the `order(id: <id>)` query, but not sure about race conditions for it with the mutation. seems like giving the client a possibility to fetch the order as return field can have some use still.

Reply:
Ok ok, I guess there's no harm in having it, we can just remove it if it's not used anymore.